internal/services/network/application_gateway_resource.go

```diff
@@ -1245,6 +1246,16 @@ func resourceApplicationGateway() *pluginsdk.Resource {
 							Default:  false,
 						},
 
+						"verify_client_cert_revocation": {
```

Comment:
We should spell out certificate here - I see that the above property has it abbreviated, this should be updated in 4.0 please add a comment above it to that effect
```suggestion
						"verify_client_certificate_revocation": {
```